docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -1024,6 +1022,26 @@ g2Client.submit("g.V()")
 The above code demonstrates how the `alias` method can be used such that the script need only contain a reference
 to "g" and "g1" and "g2" are automatically rebound into "g" on the server-side.
 
+==== RequestInterceptor
```

Comment:
might be good to have a simple example here direct in the docs or link to one in:

https://github.com/apache/tinkerpop/tree/master/gremlin-driver/src/main/java/examples 

speaking of which, are those up to date?